ledger/sandbox-classic/src/main/scala/platform/sandbox/SandboxServer.scala

```diff
@@ -91,388 +125,42 @@ object SandboxServer {
-        optWriteService = Some(new TimedWriteService(indexAndWriteService.writeService, metrics)),
-        indexService = new TimedIndexService(indexAndWriteService.indexService, metrics),
-        authorizer = authorizer,
-        engine = engine,
-        timeProvider = timeProvider,
-        timeProviderType = timeProviderType,
-        configurationLoadTimeout = config.configurationLoadTimeout,
-        initialLedgerConfiguration = Some(config.initialLedgerConfiguration),
-        commandConfig = config.commandConfig,
-        partyConfig = PartyConfiguration.default.copy(
-          implicitPartyAllocation = config.implicitPartyAllocation
-        ),
-        submissionConfig = config.submissionConfig,
-        optTimeServiceBackend = timeServiceBackendO,
-        servicesExecutionContext = servicesExecutionContext,
-        metrics = metrics,
-        healthChecks = healthChecks,
-        seedService = seedingService,
-        managementServiceTimeout = config.managementServiceTimeout,
-        enableSelfServiceErrorCodes = config.enableSelfServiceErrorCodes,
-        checkOverloaded = _ => None,
-        userManagementStore = userManagementStore,
-        ledgerFeatures = LedgerFeatures(
-          staticTime = timeServiceBackendO.isDefined,
-          commandDeduplicationFeatures = CommandDeduplicationFeatures.of(
-            Some(
-              CommandDeduplicationPeriodSupport.of(
-                offsetSupport =
-                  CommandDeduplicationPeriodSupport.OffsetSupport.OFFSET_NOT_SUPPORTED,
-                durationSupport =
-                  CommandDeduplicationPeriodSupport.DurationSupport.DURATION_NATIVE_SUPPORT,
-              )
-            ),
-            CommandDeduplicationType.SYNC_ONLY,
-            maxDeduplicationDurationEnforced = false,
-          ),
-          contractIdFeatures = ExperimentalContractIds.of(
-            v0 = ExperimentalContractIds.ContractIdV0Support.SUPPORTED,
-            v1 = ExperimentalContractIds.ContractIdV1Support.NON_SUFFIXED,
-          ),
-        ),
-      )(materializer, executionSequencerFactory, loggingContext)
-      apiServer <- new LedgerApiServer(
-        apiServicesOwner,
-        // NOTE: Re-use the same port after reset.
-        currentPort.getOrElse(config.port),
-        config.maxInboundMessageSize,
-        config.address,
-        config.tlsConfig,
-        List(
-          AuthorizationInterceptor(
-            authService,
-            userManagementStore,
-            servicesExecutionContext,
-            errorCodesVersionSwitcher,
-          )
-        ),
-        servicesExecutionContext,
-        metrics,
-      ).acquire()
-      _ <- Resource.fromFuture(writePortFile(apiServer.port))
-    } yield {
-      Banner.show(Console.out)
-      logger.withoutContext.info(
-        s"Initialized {} version {} with ledger-id = {}, port = {}, dar file = {}, time mode = {}, ledger = {}, auth-service = {}, contract ids seeding = {}{}{}",
```

Comment:
This is a more holistic presentation of the ledger than we have in the kv/app. Could you lift some o f it?

Reply:
It's all good with me if you accidentally improve the KV app logging too.

Reply:
Good point. Added it back: 336f5cb75793c4984f6e3938b81dd1e47143525e
In different PRs I'm going to improve the other runners :)

Reply:
Good point. Added it back: 336f5cb75793c4984f6e3938b81dd1e47143525e
In different PRs I'm going to improve the other runners :)